api/src/main/resources/org/openmrs/api/db/hibernate/OrderGroup.hbm.xml

```diff
@@ -70,6 +70,8 @@
                   column="uuid" length="38" unique="true" not-null="true"/>
  <many-to-one name="parentOrderGroup" class="org.openmrs.OrderGroup"
 		    not-null="false" column="parent_order_group" />
+		    <many-to-one name="orderGroupReason" class="org.openmrs.Concept"
+		    not-null="false" column="order_group_reason" />
```

Comment:
This line should be indented (see how this is done for other elements in this file).

Reply:
@ibacher i have fixed the indentions,kindly review